topk-js/index.d.ts

```diff
@@ -0,0 +1,192 @@
+/* auto-generated by NAPI-RS */
+/* eslint-disable */
+export declare class Client {
+  constructor(config: ClientConfig)
+  collections(): CollectionsClient
+  collection(name: string): CollectionClient
+}
+
+export declare class CollectionClient {
+  query(query: Query, lsn?: number | undefined | null): Promise<Array<Record<string, any>>>
+  upsert(docs: Array<Record<string, any>>): Promise<number>
+  delete(ids: Array<string>): Promise<number>
+}
+
+export declare class CollectionsClient {
+  list(): Promise<Array<Collection>>
+  create(options: CreateCollectionOptions): Promise<Collection>
+  delete(name: string): Promise<void>
+}
+
+export declare class Expr {
```

Comment:
the exprs need to be split into:
    * `SelectExpr` which contains either `LogicalExpr` or `FunctionExpr`. only accepted in `select()`
    * `FilterExpr` which contains either `LogicalExpr` or `TextExpr`. only accepted in `filter()`